packages/web-twig/src/Resources/components/Header/stories/HeaderInvertedWithActionsAndDropdown.twig

```diff
@@ -0,0 +1,92 @@
+<Header color="inverted">
+    {% include '@components/Header/stories/_partials/SpiritLogo.twig' %}
+
+    <HeaderMobileActions dialogId="header_dialog_example_3_main_navigation">
```

Comment:
Wouldn't it be easier to unify IDs with the file name instead of the stories order which may change over time?